src/test/scala/nl.knaw.dans.easy.multideposit/actions/ValidatePreconditionsSpec.scala

```diff
@@ -29,7 +29,8 @@ class ValidatePreconditionsSpec extends TestSupportFixture with BeforeAndAfterEa
 
   private val depositId = "dsId1"
   private val ldapMock: Ldap = mock[Ldap]
-  private val action = new ValidatePreconditions(ldapMock)
+  private val ffprobeMock: FfprobeRunner = mock[FfprobeRunner]
```

Comment:
This would hardly qualify as testing this new functionality! Please add tests in this class that would cover all control flows in `checkAudioVideoNotCorrupt`. Of course you can use this mocked instance to simulate the `FfprobeRunner`.

Reply:
Man! I was hoping you didn't notice :)